helm_deploy/apply-for-legal-aid/templates/cronjob-turn-off-ccms-submissions.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if eq .Values.env "production" }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-submit-ccms-off
+  labels:
+    app: {{ template "apply-for-legal-aid.name" . }}
+    chart: {{ template "apply-for-legal-aid.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  # schedule: '45 17 * * 1' # Every monday at 17:45
+  schedule: "*/5 * * * *" # Every 5 mins
```

Comment:
Should this be removed and the commented version restored?! 😄

Reply:
YES